packages/flutter/test/widgets/selection_container_test.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('SelectionContainer', () {
+    testWidgets('works', (WidgetTester tester) async {
```

Comment:
Maybe "updates its registrar and delegate based on the number of selectables" or something like that.